interface/patient_file/report/custom_report.php

```diff
@@ -484,12 +484,7 @@ function next_prev(action){
   echo '<page_header style="text-align:right;" class="custom-tag"> ' . xlt("PATIENT") . ':' . text($titleres['lname']) . ', ' . text($titleres['fname']) . ' - ' . $titleres['DOB_TS'] . '</page_header>    ';
   echo '<page_footer style="text-align:right;" class="custom-tag">' . xlt('Generated on') . ' ' . oeFormatShortDate() . ' - ' . text($facility['name']) . ' ' . text($facility['phone']) . '</page_footer>';
 
-  // Use logo if it exists as 'practice_logo.png' in the site dir
-  // old code used the global custom dir which is no longer a valid
-   $practice_logo = "$OE_SITE_DIR/images/practice_logo.png";
-   if (file_exists($practice_logo)) {
-        echo "<img src='$practice_logo' align='left'><br />\n";
-     } 
+  echo "<img src='../../../sites/default/images/practice_logo.png' align='left' style='width : 56px ; height : 90.6px'><br>";
```

Comment:
At the beginning of this file, the `require_once("../../globals.php");` allows you to use the variable $OE_SITE_DIR
Without this variable, if you have more than one /site directory, this code will be broken for any site other than /sites/default.
  Revise this to use the current site session variable above.

Reply:
okay I will try to implement that, thanks for review.